.github/workflows/set-automatic-deploys-enabled.yaml

```
@@ -1,4 +1,4 @@
-name: Set automatic_deploys_enabled (optionally image_tag too)
+name: Set automatic_deploys
```

Nit: make it human friendly to read
 
```suggestion
name: Set automatic deploys
```

---

Oh yes forgot to do that